weapons/main.json

```diff
@@ -0,0 +1,189 @@
+{
+  "$schema": "../schema/m3-weapons.schema.json",
+  "weapons": [
+    {
+      "id": 1,
+      "name": "PowerBeam",
+      "damage": 20,
+      "useRequires": ["PowerBeam"],
+      "situational": false,
+      "categories": ["All", "Beam", "UnchargedBeam"]
+    },
+    {
+      "id": 2,
+      "name": "Ice",
+      "damage": 30,
+      "useRequires": ["Ice"],
+      "situational": false,
+      "categories": ["All", "Beam", "UnchargedBeam"]
+    },
+    {
+      "id": 3,
+      "name": "Spazer",
+      "damage": 40,
+      "useRequires": ["Spazer"],
+      "situational": false,
+      "categories": ["All", "Beam", "UnchargedBeam"]
+    },
+    {
+      "id": 4,
+      "name": "Wave",
+      "damage": 50,
+      "useRequires": ["Wave"],
+      "situational": false,
+      "categories": ["All", "Beam", "UnchargedBeam"]
+    },
+    {
+      "id": 5,
+      "name": "Plasma",
+      "damage": 150,
+      "useRequires": ["Plasma"],
+      "situational": false,
+      "categories": ["All", "Beam", "UnchargedBeam"]
+    },
+    {
+      "id": 6,
+      "name": "Missile",
+      "damage": 100,
+      "useRequires": ["Missile"],
+      "shotRequires": [
+        {"cost":[
+          {"cost":[
```

Comment:
How come you use a cost object in another cost object but seemingly never use the top layer?

Reply:
Probably a copy/paste error.